content/news/playtest-20180729.md

```diff
@@ -0,0 +1,70 @@
+* Connect to private clan- or tournament-specific servers without using passwords
+* Add some flair by displaying badges that show their achievements or preferences
+
+Anonymous players continue to be identified by their IP and geo-location.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180729-profiles.png" width="600" alt="Player Profiles">
+
+Players can add badges to their profile to show their favourite mod, OS, or community achievements.
+ </div>
+
+A selection of default badges are available for selection in your forum [User Control Panel](https://forum.openra.net/ucp.php?i=-openra-openrauseraccounts-ucp-useraccounts_module&mode=profile_badges), and we will individually award badges to players who have contributed to OpenRA's development or participated in special community events. We hope to extend the profile system in the future to add more features and improve security.
+
+Other improvements to the lobby UI include an improved color chooser, the ability to select in text-fields using the mouse, and the game returning to the skirmish/mission/multiplayer panels instead of the main menu after leaving a game.
+
+Changes on the battlefield in Red Alert include:
+
+* Fixed Iron Curtain and Chronoshift effects being lost when a MCV deploys
+* Added the ability for fake structures to be built next to other fake structures
+* Kill Bounties have been removed from the campaign and disabled by default in MP
+* Tech structures and walls remain on the battlefield when their owner is defeated
+* A collection of [<span style="color: red">community balance changes</span>](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)#ra-balance-changes)
+
+Kane has also been busy, with changes in Tiberium Dawn including:
+
+* Comprehensive changes to reduce the effectiveness of APCs
+* Changes to streamline the power usage of low-tech structures
+* Reduced the amount of time that Jeep, Bike, Buggy, APC wreckage lingers on the battlefield
+* Tech structures and walls remain on the battlefield when their owner is defeated
+* A collection of other [<span style="color: red">community balance changes</span>](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)#cnc-balance-changes)
+
+<div style="text-align:center" markdown="1">
+![Chrono-vortex](/images/news/20180729-chronovortex.png)
+
+Chronoshifting MCVs opens strategic options, but beware the chrono-vortex if you cause a paradox!
+ </div>
+
+Significant work has occurred behind the scenes to fix crashes, improve performance, and extend capabilities for our modding community. [Our recent news post](/news/devblog-20180610/) discussed these topics in much more detail. We encourage modders to view the [<span style="color: red">SDK release notes</span>]() and [update instructions](https://github.com/OpenRA/OpenRAModSDK/wiki/Updating-to-a-new-SDK-or-Engine-version) for instructions on how to take advantage of these improvements.
```

Comment:
Just a reminder: The SDK notes still need to be referenced.

Reply:
Yup, its in red so that we won't miss it.